README.md

```diff
@@ -28,3 +28,31 @@ We hope the above features make this a tool that developers can use to speed up
     * ASP.NET and web development
     * [Latest .NET Core cross-platform development](https://www.microsoft.com/net/)
     * [Latest .NET SDK](https://www.microsoft.com/en-us/download/details.aspx?id=19988)
+
+### Creating an Instance of the Template (Your Own Blog)
+First, you'll need to get the [newest templating code](https://github.com/dotnet/templating).
+Switch the Branch to "rel/2.0.0-servicing." Then click the green "Clone or download" button. Copy the link in the dropdown that appears.

```

Comment:
Yeah I was going through these steps and I realized that no matter which branch you are on in GitHub, the link which you copy-paste for "git clone [link]" is the same. I guess switching branches only matters when you have cloned it/are using it yourself. Either way, it's not a huge deal whether you want to keep this or not!